scripts/create-update.ps1

```diff
@@ -0,0 +1,92 @@
+<#
```

Comment:
Administrator privileges are not required. 

It is a good idea to add a few basic checks. I'll add some to make sure MSBuild.exe and python.exe are on the path, as expect this to be a common source of frustration, so we may as well make it explicit.